### ISSUE TITLE
Bug: Save Prefs to Stuhome looking for houdini16.0 folder

### ISSUE THREAD

**Issue by AidanMerryman** [sashaouellet/SDMTools#7]
Noticed that after I cleared out my houdini16.0 prefs from my stuhome as they're no longer needed, this button stopped working. Looked at the script, changed everything to 16.5, and it was still just returning without copying. Only after I disabled the "if not" check did it actually copy over.
